src/components/Table/index.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import StoryWrapper from '../StoryWrapper';
+import { composeTable, Props } from '.';
+import { TableRow } from '../TableRow';
+
+enum FilterType {
+	name = 'name',
+	dueAt = 'dueAt',
+	createdAt = 'createdAt',
+	completed = 'completed',
+}
+
+const Table1 = composeTable<FilterType>();
+export default {
+  title: 'Table',
+  component: Table1,
+} as Meta;
+
+const Template1: Story<Props<FilterType>> = (args) => (
+  <StoryWrapper>
+    <Table1 {...args} />
+  </StoryWrapper>
+);
+
+export const Default = Template1.bind({});
+const defaultColumns = [
+	{ dataName: FilterType.name, displayName: 'NAME' },
+	{ dataName: FilterType.dueAt, displayName: 'DUE' },
+	{ dataName: FilterType.createdAt, displayName: 'CREATED' },
+	{ dataName: FilterType.completed, displayName: 'COMPLETED' },
+	{ dataName: 'actions', displayName: 'actions', hidden: true },
+]
+
+Default.args = {
```

Comment:
It doesn't look like these are type checking properly. Values can be swapped for invalid ones and TS doesn't raise an error. Maybe look into that?
It appears that `Default.bind()` returns a type of `any` instead of an appropriate type and that's why it's not raising any complaints